docs/melange-book/index.md

```diff
@@ -0,0 +1,16 @@
+# Melange Book
+
+## Motivation
+
+This is a project-based, guided introduction to Melange and its ecosystem.
+Because Melange integrates both OCaml and JavaScript ecosystems, there are a lot
```

Comment:
"integrates with"? I guess it depends on how one looks at this integration 🤔

Reply:
Replaced "integrates" with "uses"